src/main/scala/wdl4s/WdlExceptions.scala

```diff
@@ -0,0 +1,15 @@
+package wdl4s
+
+import scalaz.NonEmptyList
+
+trait ThrowableWithErrors extends Throwable {
+  def message: String
+  def errors: NonEmptyList[String]
+
+  override def getMessage = {
+    s"""$message\n${errors.list.mkString("\n")}"""
+  }
+}
+
+class UnsatisfiedInputsException(message: String) extends Exception(message)
+class ValidationException(val message: String, val errors: NonEmptyList[String]) extends Exception(message) with ThrowableWithErrors
```

Comment:
Why extend both exception and throwablewitherrors since the latter extends throwable?

Is there a case where you want the latter to not be an exception, and if not why not just make it an exception w/ errors? 

It's not a big deal either way though